### DISCUSSION TITLE
[Feature] Rich Shared Image Previews

### DISCUSSION THREAD

**Discussion by Cotywio** [immich-app/immich#2907] [Feature Request]
### The feature

It would be cool if when sharing an image we could have rich previews like Google Photos does. I believe this is related to supporting the Open Graph Protocol: https://ogp.me/

It looks like <meta property="og:image" content="xxxx> is currently set to a thumbnail which is hidden when sharing externally. 

**Difference between Immich & Google Photos in Telegram below:**
![image](https://github.com/immich-app/immich/assets/17916912/fc35d9cc-024e-4c1c-9bf0-88c8376c7e4b)

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by jrasm91** [comment]
This sounds more like a bug than anything, as we specifically added support for open graph tags originally.

**Comment by bo0tzz** [comment]
This now works again.
